package.json

```diff
@@ -55,6 +56,10 @@
 			]
 		},
 		"commands": [
+			{
+				"command": "liberty.multiStepInput",
+				"title": "Open Liberty Starter Project"
```

Comment:
Consider a more relevant command name like "liberty.starterProject".
We should also add the category "Open Liberty" (see the commands below for reference)